lib/management.js

```diff
@@ -3,12 +3,46 @@ const forge = require('node-forge');
 const arsenal = require('arsenal');
 const config = require('../conf/Config');
 const AWS = require('aws-sdk');
+const werelogs = require('werelogs');
+const bucketclient = require('bucketclient');
 
-const MetadataFileClient = arsenal.storage.metadata.MetadataFileClient;
-const mdClient = new MetadataFileClient({
-    host: config.queuePopulator.dmd.host,
-    port: config.queuePopulator.dmd.port,
+werelogs.configure({
+    level: config.log.logLevel,
+    dump: config.log.dumpLevel,
 });
+const logger = new werelogs.Logger('mongoclient');
```

Comment:
Since this is for all backends (Metadata, Mongo), we should make it more general `MetadataManager` or `Management`, I am ok with either.